packages/groq-builder/src/commands/conditional.ts

```diff
@@ -24,15 +24,15 @@ declare module "../groq-builder" {
       conditionalProjections: TConditionalProjections,
       config?: Partial<ConditionalConfig<TKey, TIsExhaustive>>
     ): ExtractConditionalProjectionResults<
-      ResultItem<TResult>,
+      ResultItem.Infer<TResult>,
```

Comment:
I like the way this self-documents - it makes it easier to understand what's happening. :+1: